lib/utils.js

```diff
@@ -85,45 +82,6 @@ export function useNow(updateEvery = 1000) {
   return now
 }
 
-export function usePostEmail() {
```

Comment:
@Evalir Are we sure we don't use this anymore? Isn't this how users register their email if they're first time jurors?

Reply:
Yep; remember this is not convert.aragon.org :) Here we don't ask for emails as we're not activating their balance. I remember asking if we would include this in the site, but decided not to at some point.

Reply:
> Yep; remember this is not convert.aragon.org :) Here we don't ask for emails as we're not activating their balance. I remember asking if we would include this in the site, but decided not to at some point.

You meant anj.aragon.org right? so we're good here? :)

Reply:
> You meant anj.aragon.org right? so we're good here? :)
Haha indeed!

Reply:
Good catch, yes this makes sense!